package.json

```diff
@@ -182,6 +182,11 @@
     "@hapi/call": "^8.0.1",
     "@hapi/subtext": "^7.0.3",
     "@kubernetes/client-node": "^0.14.3",
+    "@sentry/browser": "^6.8.0",
```

Comment:
Can we move some of these to `devDependencies`?

Reply:
no, we need these in production

Reply:
But we are bundling renderer deps with webpack...?

Reply:
yes..I guess they can be moved to `devDependencies`, I mean they are used in production, not for lint, unit tests etc.

if we are not using `yarn install --prod` (seems not, just searched), doesn't really matter to be in `devDependencies` or `dependencies`

Reply:
hmm. at least `@sentry/types` should be moved to `devDependencies`

Reply:
Remove unused `@sentry/browser` `@sentry/types` and `@sentry/utils`, not in used after https://github.com/lensapp/lens/commit/10025ba07f33a6e8ef34e4f092c38bf12a82836b

Reply:
I've just moved all sentry-related deps to `devDependencies` and it seems to work fine...

Reply:
Yes, you can do that, if preferable, we are not using `yarn install --prod` in CI as far as I know.